Makefile

```diff
@@ -63,15 +63,9 @@ build_test: jetstream_config
 build_prod: jetstream_config
 	docker build --target deploy -f app/Dockerfile -t app:deploy app/
 
-test_build: build_test
+compose_build_test: kill build_test
```

Comment:
Oh, I missed that [.circleci/config.yml executes `make test_build`](https://github.com/mozilla/experimenter/blob/84db87252deb14bd75c18f8a47702bf75ca3e33d/.circleci/config.yml#L20)